sql_generators/glean_usage/templates/dau_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,145 @@
+{{ header }}
+
+WITH base AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    -- TODO: Once this field exists we should update the reference.
+    CAST(NULL AS STRING) AS usage_profile_id,
```

Comment:
If the plan is to replace `client_id` with `usage_profile_id`, what are your thoughts about applying something like this? 
`COALESCE(client_info.usage_profile_id, client_info.client_id) AS usage_profile_id`
and using `usage_profile_id` in all subsequent queries/tables(column names)? 

I am assuming `usage_profile_id` is going to be a nested field in client_info